### PR TITLE
Allow symbols for status codes

### PR DESCRIPTION
**Pull request by dabrorius**
This is a very simple implementation of feature requested in #934.

### CODE REVIEW

**Comment by dblock** [comment]
I like it. Rebase and squash please?

**Comment by dabrorius** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 * [#936](https://github.com/intridea/grape/pull/936): Fixed default params processing for optional groups - [@dm1try](https://github.com/dm1try).
 * [#942](https://github.com/intridea/grape/pull/942): Fixed forced presence for optional params when based on a reused entity that was also required in another context - [@croeck](https://github.com/croeck).
+* [#950](https://github.com/intridea/grape/pull/950): Status method can now accept one of Rack::Utils status code symbols (:ok, :found, :bad_request, etc.) - [@dabrorius](https://github.com/dabrorius)
```
Comment (dblock) [inline CHANGELOG.md]: OCD: missing period after the line.

File: lib/grape/dsl/inside_route.rb [hunk 2]
```diff
@@ -94,7 +94,13 @@ def redirect(url, options = {})
       #
       # @param status [Integer] The HTTP Status Code to return for this request.
       def status(status = nil)
-        if status
+        if status.is_a? Symbol
+          if Rack::Utils::SYMBOL_TO_STATUS_CODE.keys.include?(status)
+            @status = Rack::Utils.status_code(status)
+          else
+            fail ArgumentError, "Status code :#{status} is not valid."
```
Comment (dblock) [inline lib/grape/dsl/inside_route.rb]: You want to specialize this error and make it localizable too, https://github.com/intridea/grape/blob/master/lib/grape/locale/en.yml.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 * [#936](https://github.com/intridea/grape/pull/936): Fixed default params processing for optional groups - [@dm1try](https://github.com/dm1try).
 * [#942](https://github.com/intridea/grape/pull/942): Fixed forced presence for optional params when based on a reused entity that was also required in another context - [@croeck](https://github.com/croeck).
+* [#950](https://github.com/intridea/grape/pull/950): Status method can now accept one of Rack::Utils status code symbols (:ok, :found, :bad_request, etc.). - [@dabrorius](https://github.com/dabrorius)
```
Comment (dblock) [inline CHANGELOG.md]: OCD: period missing after `daborius)`, not inside the phrase, see how the other lines above are :)

File: lib/grape/dsl/inside_route.rb (hunk 2 above)
Comment (dabrorius) [inline lib/grape/dsl/inside_route.rb]: This is an error that only developer should see, not the end user. Is there really need to localize it?
I used this as reference https://github.com/intridea/grape/blob/master/lib/grape/dsl/inside_route.rb#L28

**Comment by dabrorius** [comment]
@dblock I changed the error message but didn't localize it yet. As I commented before, I'm not sure if it makes sense to allow user to translate a message that API end-users (as far as I understand) will never see. If it really needs to be localized I can change it but I wanted to make sure you really want that.

**Comment by dblock** [comment]
Fair enough, merging.

**Comment by swistaczek** [comment]
Good one @dabrorius  :+1:

**Comment by dabrorius** [comment]
Thanks :)

**Pull request merged**